ngrok.yml

```diff
@@ -1,4 +1,4 @@
-authtoken: "ADD YOUR TOKEN"
+authtoken: "5nue34aMgomEjiyxPQ6rR_3m2N52Ttny8wnCtoALMEi"
```

Comment:
Probably should not have your token, nor this file in the repo.
I suggest you regenerate the token.

Reply:
God I really suck with tokens recently. I didn't even touch the file 😐

Reply:
I reset it, thanks for the headsup